src/auth.js

```diff
@@ -64,7 +64,7 @@ module.exports = app.post('/v1/login', async (req, res) => {
                 privateKeyEncoding: {
                     type: 'pkcs8',
                     format: 'pem',
-                    //                    cipher: 'aes-256-cbc', //TODO vllt machen wegen DB?
+                    //                    cipher: 'aes-256-cbc',
                     //                    passphrase: 'top secret'
```

Comment:
remove those lines as well?

Reply:
Might be a cool feature for the future because it would allow us to store them in a encrypted way in the database